extension/chrome/elements/compose-modules/compose-quote-module.ts

```diff
@@ -196,7 +196,6 @@ export class ComposeQuoteModule extends ViewModule<ComposeView> {
     $(el).remove();
     Xss.sanitizeAppend(this.view.S.cached('input_text'), this.getTripleDotSanitizedFormattedHtmlContent());
     this.tripleDotSanitizedHtmlContent = undefined;
-    this.view.inputModule.squire.focus();
```

Comment:
As long as the result is that body gets focused on reply and inputs get focused on forward, it's all good :+1:

Reply:
All is well

> body gets focused on reply and inputs get focused on forward

this is handled in https://github.com/FlowCrypt/flowcrypt-browser/pull/2985/files#diff-107b333d89353f1f2b84b7959666d8ccR251